webapp/src/components/ListItem/ListItem.vue

```diff
@@ -9,8 +9,8 @@
 <template>
   <div id="list-item">
     <h2>{{item.title}} - ({{item.votes}})</h2>
-    <button @click="upvoteItem();$emit('update-item', item)">Upvote</button>
-    <button @click="downvoteItem();$emit('update-item', item)">Downvote</button>
+    <button @click="upvoteItem();">Upvote</button>
```

Comment:
```suggestion
    <button @click="upvoteItem">Upvote</button>
```
If you give VueJS a function, VueJS will call the function for you. Here, you're calling the function when VueJS renders the template. Technically no big difference, but less code.